features/consent/src/main/res/layout/consent_details_fragment.xml

```diff
@@ -10,12 +10,20 @@
             type="in.projecteka.jataayu.core.model.Consent" />
 
         <variable
-            name="requestExpired"
-            type="Boolean" />
+            name="grantedConsentArtifact"
+            type="in.projecteka.jataayu.core.model.grantedconsent.GrantedConsentArtifactDetails" />
 
         <variable
             name="clickHandler"
             type="in.projecteka.jataayu.provider.ui.handler.ConsentDetailsClickHandler" />
+
+        <variable
+            name="requestExpiredOrGranted"
```

Comment:
It looks like the boolean (which has only 1 state) is doing two things. Can we name it something better? e.g. requestExpired